classes/01class/exercises/c01-git01/mail2kvanitha/README.md

```diff
@@ -0,0 +1,33 @@
+# c01-git01
+
+## Commands executed
+
+Commands typed to solve the "Merging" exercise
+git checkout master
+git merge branch-name
+git checkout branch-name
+git merge master
+
+```
+
+Commands typed to solve the "Rebasing" exercise
+git checkout -b branch-name
```

Comment:
that's correct, the exercise uses `bugFix` for the `branch-name` but the result is the same 👍